### PR TITLE
some more cassandra tweaking

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
A few more things from http://docs.datastax.com/en/landing_page/doc/landing_page/recommendedSettings.html applied

**Comment by ktoso** [comment]
Most notably turning off swap

File: infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb [hunk 1]
```diff
@@ -49,3 +49,19 @@
...
+
+bash 'zone_reclaim_mode' do
+  code <<-EOH
+    echo 0 > /proc/sys/vm/zone_reclaim_mode
+  EOH
+end
+
+bash "disable swap" do
+  code <<-EOH
+    swapoff --all
```
Comment (patriknw) [inline infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb]: should we do this on all nodes?

Comment (ktoso) [inline infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb]: probably good idea, adding

File: infra/site-cookbooks/akka-linux-tweaks/recipes/default.rb [hunk 2]
```diff
@@ -18,3 +18,9 @@
     sysctl net.core.wmem_max=2097152
   EOH
 end
+
+bash "disable swap" do
+  code <<-EOH
+    swapoff --all
+  EOH
+end
```
Comment (ktoso) [inline infra/site-cookbooks/akka-linux-tweaks/recipes/default.rb]: disabling swap added to akka nodes // @patriknw 
(change being rolled out as we speak)

**Pull request merged**